fbw-a380x/src/systems/shared/src/OperatingAltitudes.tsx

```diff
@@ -10,7 +10,7 @@ export class A380AltitudeUtils {
    */
   static calculateRecommendedMaxAltitude(gw: number, disa: number, round = 500) {
     return MathUtils.round(
-      54344 -
+      52442 -
```

Comment:
This is intentional correct?

Reply:
Indeed, the REC MAX is updated inline with the new performance numbers.